pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunction.java

```diff
@@ -49,6 +51,13 @@
    */
   String getResultColumnName();
 
+  /**
+   * Returns a list of input expressions needed for performing aggregation.
+   *
+   * @return List of input expressions
```

Comment:
(nit) We don't have return argument documented in this file, maybe keep it consistent?

Reply:
Ok, will remove. It does not enough value to go out of the norm I guess.